src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java

```diff
@@ -64,6 +65,57 @@
     private long nextTimeToParse = 0;
     private boolean savedSinceLastParsing = false;
 
+    /**
+     * Checks that all required keys exist in the incoming json object.
+     *
+     * @param JsonObject instance
+     * @return boolean
+     * */
+    public boolean hasRequiredJsonKeys(JsonObject instance) {
+        if (instance.has(HOST) && instance.has(PORT) && instance.has(NAME)
+                && instance.has(CONTEXT_PATH) && instance.has(HTTPS)) {
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Checks if json values in backend instance are null.
+     *
+     * @param JsonObject instance
+     * @return boolean
+     * */
+    public boolean containsNullValues(JsonObject instance) {
+        for (Map.Entry<String, JsonElement> e : instance.entrySet()) {
+            if (e.getValue().isJsonNull() || e.getValue() == null) {
+                LOG.debug(e.toString() + " can not be null!");
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Checks that the incoming json data does not contain additional keys. Any
+     * unrecognized keys are logged.
+     *
+     * @param JsonObject instance
+     * @return boolean
+     * */
+    public boolean containsAdditionalKeys(JsonObject instance) {
+        if (instance.size() > ALLOWED_JSON_KEYS.size()) {
+            for (Map.Entry<String, JsonElement> e : instance.entrySet()) {
```

Comment:
Please do not use single letter variables.

Reply:
Fixed.